spec/ADAS/ADAS.vspec

```diff
@@ -200,6 +200,45 @@ ESC.IsEngaged:
   type: sensor
   description: Indicates if ESC is currently regulating vehicle stability. True = Engaged. False = Not Engaged.
 
+ESC.IsStrongCrossWindDetected:
+  datatype: boolean
+  type: sensor
+  description: Indicates if ESC system is detecting strong cross winds.
+               True = Strong Cross winds detected.
```

Comment:
Other descriptions provide information for False as well. Granted, this is a bit redundant, but not providing it seems inconsistent (see line 201 above).